gig-marketplace.md

```diff
@@ -0,0 +1,57 @@
+## Title
+
+* Gig Marketplace
+
+## Patlet
+
+InnerSource projects at your company are finding it difficult to attract contributors.  
+
+Establish a marketplace by creating an intranet website that list specific InnerSource project needs as "gigs" with explicit time and skill requirements.
+This will enable managers to better understand their employee’s time commitment and professional benefits thereby increasing the likelihood of garnering approval to make InnerSource contributions.
+
+## Problem
+
+There is no single, uniform way for InnerSource project owners and potential contributors to discover each other. 
+It is difficult for employees to communicate to their management what commitment of time they will need to make to an InnerSource project and how they will benefit.  
+Managers have no uniform way of keeping track of or rewarding their employee’s involvement in InnerSource projects.
+
+## Story
+ 
+You’ve successfully created an InnerSource program at your company and have buy-in from senior management, middle-management and developers.  However, after nearly a year there have been few actual contributions made to any InnerSource projects outside of the teams that originally created them.  After interviewing all the parties involved the main sticking point seems to be that it is difficult to know what the exact level of commitment developers will be asked to make if they choose to get involved in an InnerSource project. There is also no uniform way of advertising what opportunities for contributors exist, what they’ll be asked to do and approximately how long it might take.  Managers are supportive and want their employees to participate but so far have lacked a way of documenting and accounting for their employees’ activities within InnerSource projects.  What can be done to improve this situation for all the parties involved (InnerSource project owners, potential contributors and development managers)? 
+
+## Forces
```

Comment:
I feel like some of these **Forces** are written as lack-of-the-solution.  I'd advocate for thinking of what is the underlying force that causes a solution to be needed and write _that_ as the **Force**.

e.g. instead of

> `It is difficult for managers to approximate what the time/effort commitment will be for employees taking on InnerSource project tasks.`

something like

> `Managers need to plan and account for employees time commitments`

or instead of

> `The less insight a manager has into the effort and accomplishment of their employees in InnerSource projects, the more unlikely it is that he can reward their employees for their InnerSource work.`

something like

> `Managers and employees both want inner source contributions to be rewarded`.

Reply:
Great point.  Will go back and rephrase these to follow this more succinct phrasing.